app/api/auth/individual/login/route.ts

```diff
@@ -40,6 +43,11 @@ export async function POST(request: Request) {
 
     const { user_id } = user;
 
+    if (!user.verified)
+      return NextResponse.redirect(
+        new URL(`/verify/individual/${user_id}`, request.url)
+      );
+
     return res.json({
```

Comment:
If you're using NextResponse as res, plus move the status from the initial JSON object to a separate one:
return res.json({message: "Login successful"}, {status: 201})

That way we can have access to the ok property in fetch callback